spec/unit/settings/ini_file_spec.rb

```diff
@@ -56,6 +56,7 @@
     end
 
     expect(config_fh.string).to eq(<<-CONF)
+[main]
 name = value
```

Comment:
The indentation for these two lines doesn't match the indentation for all the other settings file heredocs